lib/node/rpc.js

```diff
@@ -2366,7 +2366,7 @@ class RPC extends RPCBase {
 
     try {
       const res = Resource.decode(ns.data);
-      return res.toJSON();
```

Comment:
This change is required to correctly render the JSON response with the `name` field filled.

Reply:
TODO: break this into its own PR

Reply:
Merged in https://github.com/handshake-org/hsd/commit/a90a45bb9ea2c267e47bbaa173da1f181123f35f